dashboard/tests/conftest.py

```diff
@@ -1 +1,13 @@
+import os
+import pytest
 from ray.tests.conftest import *  # noqa
+
+
+@pytest.fixture
+def enable_test_module():

```

Comment:
One thing you could also do here is change this to a statement that cleans itself up. For instance:
```
@pytest.fixture(scope="module")
def disable_test_module():
    os.environ.pop("RAY_DASHBOARD_MODULE_TEST", None)
    yield
    os.environ["RAY_DASHBOARD_MODULE_TEST"] = "true"
```
which would automatically re-enable tests after the module exits.

Then I think you would only need one of these two fixtures.

Reply:
Good idea. The test module is disabled by default, so one `enable_test_module()` is enough for the tests. I will merge the `disable_test_module()` and `enable_test_module()`.